Dev automation

Ultimate-Storm opened the pull request.

kevinxpfeiffer commented on workspace/automate_scripts/server_setup/gpu_env_setup.sh:
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -eux
+
+#echo "Ekfz_swarm_2" | sudo -S su
```
Shouldn't we delete our password

Ultimate-Storm replied:
Yes we should

Ultimate-Storm commented on workspace/automate_scripts/launch_swarm_learning/gen_cert.sh:
```diff
@@ -0,0 +1,25 @@
+fi
+
+cp -r workspace/swarm_learning_scripts/gen-cert workspace/"$workspace"/
+sudo ./workspace/"$workspace"/gen-cert -e "$workspace" -i "$ip_addr"
```
Yes it's not a problem any more

kevinxpfeiffer commented on workspace/automate_scripts/server_setup/install_containers.sh:
```diff
@@ -0,0 +1,21 @@
+set -eux
+
+echo "Login to hub.myenterpriselicense.hpe.com"
+sudo docker login -u jiefu.zhu@tu-dresden.de -p hpe_eval hub.myenterpriselicense.hpe.com
```
Do we want to maybe create an extra gmail address for swarm learning and register it with hpe so we can leave personal email out here?

Ultimate-Storm replied:
It's ok to leave mine here since we will be using this email address also as IT support for our whole kathlab repo

oliversaldanha25 replied:
Yes I agree we should do this. @kevinxpfeiffer Could you create one Gmail with katherlab_swarm as the name?

kevinxpfeiffer replied:
Yes I do it immediately

kevinxpfeiffer reviewed (approved):
Good job Jeff! 

I have added a few notes directly to the code location.

In the future, maybe we can create a .gitignore file and not upload the certificate with it.

When we have successfully completed the automation and our project is running safely, we can perhaps remove the workspaces that we no longer need.

If you have tested your scripts on the hosts and it all worked, you are welcome to merge it :)

Ultimate-Storm commented:
> Good job Jeff!
> 
> I have added a few notes directly to the code location.
> 
> In the future, maybe we can create a .gitignore file and not upload the certificate with it.
> 
> When we have successfully completed the automation and our project is running safely, we can perhaps remove the workspaces that we no longer need.
> 
> If you have tested your scripts on the hosts and it all worked, you are welcome to merge it :)

Thanks for your CR, I have added the certificates to gitignore also.
